client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-card/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { Card, Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { useState, ReactElement } from 'react';
+import SiteErrorContent from '../site-error-content';
+import SiteStatusContent from '../site-status-content';
+import type { ReactChild } from 'react';
+
+import './style.scss';
+
+interface Props {
+	rows: {
+		site: { value: { blog_id: number; url: string }; error: string; type: string; status: string };
+		scan: { threats: number };
+		plugin: { updates: number };
+		[ key: string ]: any;
+	};
+	columns: {
+		site: ReactChild;
+		backup: ReactChild;
+		scan: ReactChild;
+		monitor: ReactChild;
+		plugin: ReactChild;
+		[ key: string ]: any;
+	};
+}
+
+export default function SiteCard( { rows, columns }: Props ): ReactElement {
+	const [ isExpanded, setIsExpanded ] = useState( false );
+
+	const toggleIsExpanded = () => {
+		setIsExpanded( ! isExpanded );
+	};
```

Comment:
If new state depends upon previous state, it's better to use [Functional Updates](https://reactjs.org/docs/hooks-reference.html#functional-updates).

```suggestion
	const toggleIsExpanded = () => {
		setIsExpanded( ( prevValue ) => ! prevValue );
	};
```